recipes/necat/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = "0.0.1_update20200803" %}
+{% set sha256 = "5ddd147b5be6b1fac2f6c10b18c9b587838f2304d2584087c4ed6f628eced06c" %}
+
+package:
+  name: necat
+  version: {{ version }}
+
+source:
+  url: https://github.com/xiaochuanle/NECAT/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - tool_main.mk.patch
+    - src_test_main.mk.patch
+    - src_makedb_main.mk.patch
+    - src_pairwise_mapping_main.mk.patch
+    - src_pm_one_volume_main.mk.patch
+    - src_partition_candidates_main.mk.patch
+    - src_consensus_main.mk.patch
+    - src_sequence_length_stats_main.mk.patch
+    - src_split_long_reads_main.mk.patch
+    - src_asm_pm_asmpm.mk.patch
+    - src_partition_m4_main.mk.patch
+    - src_trim_bases_pm4.mk.patch
+    - src_reference_mapping_main.mk.patch
+    - src_trim_bases_largest_cover_range.mk.patch
+    - src_trim_bases_trim_bases.mk.patch
+    - src_trim_bases_order_results.mk.patch
+    - src_trim_bases_extract_trimmerd_reads.mk.patch
+    - src_trim_bases_accurate0_pm4.mk.patch
+    - src_trim_bases_accurate0_largest_cover_range.mk.patch
+    - src_trim_bases_accurate0_trim_bases.mk.patch
+    - src_trim_bases_accurate_trim_bases.mk.patch
+    - src_trim_bases_accurate_pm4.mk.patch
+    - src_trim_bases_accurate_largest_cover_range.mk.patch
+    - src_extract_sequences_extract_sequences.mk.patch
+    - src_renumber_sequences_main.mk.patch
+    - src_reorder_cns_reads_main.mk.patch
+    - src_ctg_cns_main.mk.patch
+    - src_ctgpm_ctgpm.mk.patch
+    - src_ctgpm_fix_can_info.mk.patch
+    - src_ctgpm_split_ctgs.mk.patch
+    - src_pm4_main.mk.patch
+    - src_preprocess_raw_reads_main.mk.patch
+    - src_reference_mapping_rm_one_vol_main.mk.patch
+    - src_fsa_fsa.mk.patch
+    - src_fsa_assemble.mk.patch
+    - src_fsa_bridge.mk.patch
+    - src_fsa_filter.mk.patch
+    - src_fsa_rd_extract.mk.patch
+    - src_fsa_rd_stat.mk.patch
+    - src_fsa_rd_tools.mk.patch
+
+build:
+  skip: True  # [osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - zlib
+  run:
+    - perl
+    - perl-env
+
+test:
+  commands:
+    - necat
```

Comment:
pipe to grep to ensure expected output ie `necat | grep 'some expected output'
grep will return non-zero if it is not found

Reply:
done